src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -506,6 +506,19 @@ struct InitializeJ : db::SimpleTag {
   }
 };
 
+// Tags that generates an `Cce::InitializeJ::InitializeJ` derived class from an
+// analytic solution.
+struct AnalyticInitializeJ : InitializeJ {
```

Comment:
Still not supported and should be avoided. It might technically work, but is one of this "If it breaks we will probably just disable the code that uses that misfeature". I don't think there's a way to guard against it either, unfortunately. This is generally what "pure base tags" were meant to solve. Basically, simple tags can inherit from base tags only, compute tags must inherit from simple tags.

Reply:
In the fixup, I introduce a pure-base tag and inherit from it in the two separate cases -- I agree the inheriting from another simple tag is bad practice.

Reply:
:+1: sounds good :) Sorry, was just scrolling through the comments and realized there was a sort of ping to me here